radio/src/gui/9X/menu_model_telemetry.cpp

```diff
@@ -606,7 +606,7 @@ void menuModelTelemetry(uint8_t event)
     switch (k) {
 #if defined(CPUARM)
       case ITEM_TELEMETRY_PROTOCOL_TYPE:
-        g_model.telemetryProtocol = selectMenuItem(TELEM_COL2, y, STR_TELEMETRY_TYPE, "\017FrSky S.PORT\0  FrSky D\0       FrSky D (cable)", g_model.telemetryProtocol, PROTOCOL_TELEMETRY_FIRST, CASE_PCBSKY9X(PROTOCOL_FRSKY_D_SECONDARY) attr, event);
+        g_model.telemetryProtocol = selectMenuItem(TELEM_COL2, y, STR_TELEMETRY_TYPE, "\017FrSky S.PORT\0  FrSky D\0       FrSky D (inv)\0 FrSky D (cable)", g_model.telemetryProtocol, PROTOCOL_TELEMETRY_FIRST, CASE_PCBSKY9X(PROTOCOL_FRSKY_D_SECONDARY) attr, event);
```

Comment:
There is no hard rule, but a compromise between the limited resources and timing... We don't have enough devs to have branch maintainers who backport things or otherwise do work twice so we try to avoid. So for some time after a major release we'll continue working in `master`, until `next` starts having too many differences to be able to be regularly rebased on current master and thus needs duplication of commits like now. Anything that changes eeprom storage format needs to be done in next, so that pretty much decides when it happens.

Making an "Invert serial port" checkbox would probably be better in the long run indeed, but the ability to do that now on `master` precisely depends on whether we have a free bit in the model data structure to store that setting. If not, then doing it this way would need to target `next`

Of course we want to avoid including "temporary behaviors" where the same thing gets done differently in 2 successive versions to avoid confusing users, so we don't want to have it done with a new protocol choice now, and then change to the independent checkbox in 2.2.